Simplenote/src/main/java/com/automattic/simplenote/NoteMarkdownFragment.java

```diff
@@ -162,7 +162,9 @@ protected void onPostExecute(Void nada) {
             mIsLoadingNote = false;
 
             if (mNote != null) {
-                getActivity().invalidateOptionsMenu();
+                if(getActivity() != null) {
```

Comment:
```suggestion
                if (getActivity() != null) {
```
Just need a space there ;) Looks good otherwise.

Reply:
Sorry about that!